### ISSUE TITLE
sort order in Android TV app

### ISSUE THREAD

**Issue by moreje** [jellyfin/jellyfin-androidtv#154]
Seems that in Android TV app, you can not reverse the sort order of movies, there is no option..

**Comment by moreje** [comment]
hello, it looks loke this feature is still missing....
sorting is working fine on Web app...but really bugged on Android TV app :(
is ther plans to improve this?

**Comment by nielsvanvelzen** [comment]
We are planning to improve it but it takes some time

**Comment by moreje** [comment]
happy to read that you are working on it!

**Comment by tranis** [comment]
any ETA? or anyone know a work around, as currently the order seems to be extremely unsorted...

**Comment by nielsvanvelzen** [comment]
No progress on this issue has been made yet so I can't give any ETA.

**Comment by kylepwarren** [comment]
> any ETA? or anyone know a work around, as currently the order seems to be extremely unsorted...

Yeah, the sort definitely needs some work. **browse by letter seems like the best way to browse a large library so far.**

**Comment by s-h-a-d-o-w** [comment]
Would like to mention another use case: Playing archived videos in their chronological order.
Currently, obviously only backwards or going one by one manually is possible. A bit frustrating with dozens of videos. 😅

**Comment by PororoManon** [comment]
Any updates on these issues? I have file like this, and Jellyfin shows the title only, so Its show alphabetically which not match by episodes. I can sort it by date added, but I need to scroll to right or bottom before play the files. Thanks! :D
![image](https://github.com/jellyfin/jellyfin-androidtv/assets/148423664/f8742108-bb09-44f2-8c7f-2d3d55a57d5b)

**Comment by BigBoySmoke-69** [comment]
As this is still open is there any plans to enable toggle for this? It's really jarring seeing things on the mobile and windows apps/browser sorted release date (oldest first) and then this being only newest first.
Makes it a really inconsistent experience.
